static/app/components/feedback/feedbackItem/crashReportSection.tsx

```diff
@@ -0,0 +1,52 @@
+import styled from '@emotion/styled';
+
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import EventOrGroupExtraDetails from 'sentry/components/eventOrGroupExtraDetails';
+import EventOrGroupHeader from 'sentry/components/eventOrGroupHeader';
+import Section from 'sentry/components/feedback/feedbackItem/feedbackItemSection';
+import {IconIssues} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {Event, Group, Organization} from 'sentry/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+
+interface Props {
+  crashReportId: string;
+  organization: Organization;
+  projSlug: string;
+}
+
+export default function CrashReportSection({
+  crashReportId,
+  organization,
+  projSlug,
+}: Props) {
+  const eventEndpoint = `/projects/${organization.slug}/${projSlug}/events/${crashReportId}/`;
+  const {data: crashReportData} = useApiQuery<Event>([eventEndpoint], {staleTime: 0});
+  // need to fetch the issue data since the event data is not complete enough for the component
```

Comment:
Do we have an endpoint that fetches an issue from an event id?

Reply:
not to my knowledge, this is what's documented :( 
![image](https://github.com/getsentry/sentry/assets/56095982/4bffcb15-20ea-44f2-a7d5-13c5c0cd2e81)